website/docs/r/vcenter.html.markdown

```diff
@@ -0,0 +1,76 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_vcenter"
+sidebar_current: "docs-vcd-resource-vcenter"
+description: |-
+  Provides a resource to manage vCenters.
+---
+
+# vcd\_nsxt\_vcenter
+
+Provides a resource to manage vCenters.
+
+~> Only `System Administrator` can create this resource.
+
+## Example Usage
+
+```hcl
+resource "vcd_vcenter" "test" {
+  name                   = "TestAccVcdTmVcenter-rename"
+  url                    = "https://host:443"
+  auto_trust_certificate = true
+  refresh_on_read        = true
+  username               = "admim@vsphere.local"
+  password               = "CHANGE-ME"
+  is_enabled             = true
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) A name for vCenter server
+* `description` - (Optional) An optional description for vCenter server
+* `username` - (Required) A username for authenticating to vCenter server
+* `password` - (Required) A password for authenticating to vCenter server
+* `refresh_on_read` - (Optional) An optional flag to trigger refresh operation on every read. This
+  might take some time, but can help to load up new artifacts. Update is a no-op. Default `false`.
```

Comment:
Need an explanation of what artefacts. Alternatively, a rename of "artefacts".

Reply:
I added at least one component (Supervisors). I don't think this is a finite list, but generally - it should reload vCenter exposed artifacts